src/applications/static-pages/mhv-signin-cta/components/messages/UnverifiedAlert.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+// eslint-disable-next-line import/no-named-default
+import { default as recordEventFn } from '~/platform/monitoring/record-event';
+import {
+  SERVICE_PROVIDERS,
+  CSP_IDS,
+} from '~/platform/user/authentication/constants';
+import CustomAlert from './CustomAlert';
+
+export const headingPrefix = 'Verify your identity';
+
+/**
+ * Alert to show a user that is not verified (LOA1).
+ * @property {*} recordEvent the function to record the event
+ * @property {string} serviceDescription optional description of the service that requires verification
+ * @property {string} signInService the ID of the sign in service
+ */
+const UnverifiedAlert = ({
+  recordEvent = recordEventFn,
+  serviceDescription,
+  signInService = CSP_IDS.ID_ME,
+}) => {
+  const signinServiceLabel = SERVICE_PROVIDERS[signInService]?.label;
+  const headline = serviceDescription
+    ? `${headingPrefix} to ${serviceDescription}`
+    : headingPrefix;
+
+  const defaultAlert = () => {
+    return (
+      <CustomAlert
+        headline={headline}
+        icon="lock"
+        status="warning"
+        recordEvent={recordEvent}
+      >
+        <div>
+          <p>
+            We need you to verify your identity for your{' '}
+            <strong>{signinServiceLabel}</strong> account. This step helps us
+            keep your information safe and prevent fraud and identity theft.
+          </p>
+          <p>
+            This one-time process often takes about 10 minutes. You’ll need to
+            provide certain personal information and identification.
+          </p>
+          <p>
+            <a
+              className="vads-c-action-link--green"
+              href="/resources/verifying-your-identity-on-vagov/"
+              hrefLang="en"
+            >
+              Verify your identity with {signinServiceLabel}
+            </a>
+          </p>
+        </div>
+      </CustomAlert>
+    );
+  };
+
+  const mhvAlert = () => {
```

Comment:
Typically, the naming convention for functions that return jsx/react is `CapitalCase`. Suggesting `mhvAlert` should be named `MhvAlert`

Reply:
Fixed. Thx.